src/core/comm/mpi_comm.cc

```diff
@@ -69,10 +69,8 @@ MPINetwork::MPINetwork(int argc, char* argv[])
   int provided, init_flag = 0;
   CHECK_MPI(MPI_Initialized(&init_flag));
   if (!init_flag) {
-    log_coll.fatal(
-      "MPI has not been initialized, it should be initialized by "
-      "the networking backend.");
-    LEGATE_ABORT;
+    MPI_Init_thread(0, 0, MPI_THREAD_MULTIPLE, &provided);
```

Comment:
Maybe add a `log_coll.info(...)` to mention MPI is initialized by legate. Mainly for debugging. 
Could you please also update line 107 to use the log_coll.info. Sorry, it was a mistake of mine.

Reply:
https://github.com/nv-legate/legate.core/pull/711/commits/6b8a8c3d11d4d7cf3c0bcd6df378aaef39a35231